doc/interfaces/registration_flow.rst

```diff
@@ -0,0 +1,165 @@
+.. _registration_flow:
+
+============
+Registration
+============
+
+This tutorial walks through the steps to perform image-based and
+streamline-based registration using DIPY. Multiple registration methods are
+available in DIPY.
+
+You can try these methods using your own data; we will be using the data in
+DIPY. You can check how to :ref:`fetch the DIPY data<data_fetch>`.
+
+------------------
+Image Registration
+------------------
+
+DIPY's image registration workflow can be used to register a moving image to a
+static image by applying different transformations, such as center of mass,
+translation, and rigid body or full affine (including translation, rotation,
+scaling and shearing) transformations. During such a registration process, the
+static image is considered to be the reference, and the moving image is
+transformed to the space of the static image. Registration methods use some
+sort of optimization process, and a given metric or criterion (like maximizing
+the mutual information between the two input images) that is optimized during
+the process, to achieve the goal.
+
+The DIPY image registration workflow applies the specified type of
+transformation to the input images, and hence, users are expected to choose the
+type of transformation that best matches the requirements of their problem.
+Alternatively, the workflow allows to perform registration in a progressive
+manner. For example, using affine registration with ``progressive`` set to
+``True`` will involve center of mass, translation, rigid body and full affine
+registration. When ``progressive`` is ``False`` the registration will include
```

Comment:
(...) registration; meanwhile, if {double_backticks}progressive{double_backticks} is set to {double_backticks}False{double_backticks} for an affine registration, it will include (...)

Reply:
Addressed in 949b676